src/main/java/colress/task/Deadline.java

```diff
@@ -0,0 +1,55 @@
+package colress.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents the Deadline task, which contains a String field representing the description, a boolean field reflecting
+ * whether the task is marked as done or not and a LocalDate field representing its deadline.
+ */
+public class Deadline extends Task {
+    private final LocalDate DEADLINE;
```

Comment:
I noticed you have a lot of your attribute names in all caps, but to my understanding attributes like deadline don't qualify as constants

Reply:
Changed to lowercase. I wrongly characterised the variables as constants .Thanks for the feedback👍.